### PR TITLE
Improve creation of RW connections database

### PR DESCRIPTION
**Pull request by andre8244**
Create connections table if connections database does not contain it.
This happens if connections database is read before its has been created.

NethServer/dev#6242

### CODE REVIEW

File: root/etc/e-smith/events/actions/nethserver-openvpn-create-connections-db [hunk 1]
```diff
@@ -22,18 +22,37 @@
...
 
-	c.execute("CREATE INDEX idx_common_name ON connections (common_name)")
+    if output and output[0] == 'connections':
+        table_exists = True
 
-	conn.commit()
-	conn.close()
+    if not table_exists:
+        create_connections_db()
+        syslog.syslog('Created roadwarrior connections database {}'.format(DATABASE_PATH))
```
Comment (gsanchietti) [inline root/etc/e-smith/events/actions/nethserver-openvpn-create-connections-db]: Just output on stdout and the output will be sent to the log.

File: root/etc/e-smith/events/actions/nethserver-openvpn-create-connections-db [hunk 2]
```diff
@@ -23,17 +23,35 @@
...
 database_exists = os.path.isfile(DATABASE_PATH)
 
 if not database_exists:
-	conn = sqlite3.connect(DATABASE_PATH)
-	c = conn.cursor()
-
-	c.execute('''CREATE TABLE connections
-			(common_name text, virtual_ip_addr text, remote_ip_addr text, start_time integer, duration integer, bytes_received integer, bytes_sent integer)''')
+    create_connections_db()
+    print('Created roadwarrior connections database {}'.format(DATABASE_PATH))
```
Comment (gsanchietti) [inline root/etc/e-smith/events/actions/nethserver-openvpn-create-connections-db]: This is the convention for output during the events
```suggestion
    print('[NOTICE] Created roadwarrior connections database {}'.format(DATABASE_PATH))
```

**Review by edospadoni — APPROVED** [review]

File: root/etc/e-smith/events/actions/nethserver-openvpn-create-connections-db [hunk 3]
```diff
@@ -23,17 +23,35 @@
...
 
-	c.execute("CREATE INDEX idx_common_name ON connections (common_name)")
+    if output and output[0] == 'connections':
+        table_exists = True
 
-	conn.commit()
-	conn.close()
+    if not table_exists:
+        create_connections_db()
+        print('Created roadwarrior connections database {}'.format(DATABASE_PATH))
```
Comment (gsanchietti) [inline root/etc/e-smith/events/actions/nethserver-openvpn-create-connections-db]:
```suggestion
        print('[NOTICE] Created roadwarrior connections database {}'.format(DATABASE_PATH))
```

**Review by gsanchietti — CHANGES_REQUESTED** [review]

**Review by gsanchietti — APPROVED** [review]
Ok! Squash & merge ;)

**Pull request merged**